ONSdigital/census-rm-ddl
# Tidying up DDL scripts, combining groundzero scripts and removing unnecessary scripts

# Motivation and Context
Updated groundzero scripts to match current prod environment

# What has changed
Updated main groundzero scripts and removed unnecessary scripts from repository 

# How to test?
Wipe database/schemas and recreate in dosql-i-know-what-i-am-doing and add:

- name: SPRING_JPA_HIBERNATE_DDL_AUTO
   value: validate 

...to bottom of census-rm-kubernetes - action-scheduler-deployment.yml/uac-qid-service-deployment.yml/case-processor-deployment.yml then redeploy these.

From census-rm-kubernetes/optional, redeploy notify-stub-deployment.yml/notify-stub-service.yml so that these appear in list of pods in GCP environment then run acceptance tests

# Links
https://trello.com/c/nQOfMX9w/353-update-groundzero-script-to-create-database-schema8
